bucket.tf

```diff
@@ -1,7 +1,7 @@
 data "aws_caller_identity" "current" {}
 
 resource "aws_s3_bucket" "build_cache" {
-  bucket = "${data.aws_caller_identity.current.account_id}-gitlab-runner-cache-test"
+  bucket = "${data.aws_caller_identity.current.account_id}-${var.environment}-cache"
```

Comment:
maybe better to name `${data.aws_caller_identity.current.account_id}-${var.environment}-gitlab-cache`?

Reply:
any comments?
My propose caused to help AWS Console users to explicitly identify bucket purpose.

Reply:
Agree. Done.